mutt_config.c

```diff
@@ -164,7 +164,7 @@ static struct ConfigDef MainVars[] = {
   { "ask_cc", DT_BOOL, false, 0, NULL,
     "Ask the user for the carbon-copy recipients"
   },
-  { "assumed_charset", DT_STRING, 0, 0, charset_validator,
+  { "assumed_charset", DT_SLIST|SLIST_SEP_COLON|SLIST_ALLOW_EMPTY, 0, 0, charset_validator,
```

Comment:
As you mentioned on IRC, the validator needs updating.
Calling _this_ validator will probably crash (it'll be passed `struct Slist*`).
Let's set it to `NULL`, for now.

```suggestion
  { "assumed_charset", DT_SLIST|SLIST_SEP_COLON|SLIST_ALLOW_EMPTY, 0, 0, NULL,
```